recipe/meta.yaml

```diff
@@ -20,7 +23,7 @@ source:
   - path: cmake/  # [win]
 
 build:
-  number: 2
+  number: 3
```

Comment:
```suggestion
  number: 0
```

Reply:
I incremented rather than reset the build number because the other packages have the same versions and hashes, and so if we don't bump the build number we'll have conflicts for aprutil and apriconv.